src/_P016_IR.ino

```diff
@@ -382,10 +382,9 @@ boolean Plugin_016(uint8_t function, struct EventStruct *event, String& string)
           }
 
           const String P016_HEX_INPUT_PATTERN = F("(0x)?[0-9a-fA-F]{0,16}"); // 16 nibbles = 64 bit, 0x prefix is allowed but not added by
-                                                                             // default
-          addRowLabel(F("Code - command map"));
+                                                                             // defaul
 
-          html_table(EMPTY_STRING);
+          html_table("normal");
```

Comment:
Also, when using strings, try to wrap them in a `F()` macro.
```c++
html_table(F("normal"));
```

Flash strings will be dealt with directly from flash.
Non-flash strings will be placed in "program memory" and thus take up RAM. (twice if they are actually used)